docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -93,21 +95,29 @@ To reduce your JavaScript bundle size, follow the steps below.
 
 #### Step 1: Profile your bundle
 
-The first step to fix this is to figure out what's going on. Use `gatsby-plugin-webpack-bundle-analyzer` to profile your bundle. When you add this tool to your `gatsby-config.js`, you can analyze the bundle size on each page in your application.
+The first step to fix this is to figure out what's going on. Use `gatsby-plugin-webpack-bundle-analyzer` and the _experimental_ plugin `gatsby-plugin-perf-budgets` to profile your bundle. When you add this tool to your `gatsby-config.js`, you can analyze the bundle size on each page in your application.
```

Comment:
`gatsby-plugin-webpack-bundle-analyzer` is deprecated and people should use `gatsby-plugin-webpack-bundle-analyzer-v2`

Reply:
Good catch!

Reply:
Fixed through the doc.